errors/parallel-build-without-worker.mdx

```diff
@@ -0,0 +1,20 @@
+---
+title: Parallel Build Without Build Worker
+---
+
+## Why This Error Occurred
+
+The `experimental.parallelServerAndEdgeCompiles` and `experimental.parallelServerBuildTraces`
```

Comment:
Rename to `parallelServerCompiles`? "nodejs" and "edge" are runtime but are both "server" target

Reply:
Done.